tabs/src/components/SendPayment.tsx

```diff
@@ -98,21 +76,13 @@ const SendPayment: React.FC<SendPopupProps> = ({
 
   const handleSendClick = () => {
     setIsLoading(true);
-    const memo = sendAnonymously ? 'Anonymous Payment' : 'Payment'; // Adjust memo based on user selection
-    console.log('handleSendClick. Start. Invoice link: ', invoice);
+    const memo = 'Payment'; // Static memo (no anonymous option in this version)
```

Comment:
The definition of "anonymous" is for a zap where the `extra { from: {} }` is left blank. Doesn't really apply here unless we allow for sending of zaps in this pop-up, which we don't, that's done from the allowance wallet (which has no direct send function). You should still be able to add a memo rather than a fixed string. Needs to really be a field in pop-up in the UX.